src/Microsoft.ML.Data/Transforms/GenerateNumberTransform.cs

```diff
@@ -163,7 +163,7 @@ public static Bindings Create(ModelLoadContext ctx, Schema input)
                 return new Bindings(useCounter, states, input, false, names);
             }
 
-            public void Save(ModelSaveContext ctx)
+            internal void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 12, length = 8)

This is on the private class `Bindings`, right? There's no need I think to make this internal.

Reply:
Sounds good, I will revert here.

---
In reply to: [254486483](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254486483) [](ancestors = 254486483)

Reply:
it's still internal

---
In reply to: [254568639](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254568639) [](ancestors = 254568639,254486483)